doc/RELEASE.md

```diff
@@ -0,0 +1,24 @@
+# Anon Release
+
+## Installing Debian/Ubuntu Package
+
+1. Download package. Package name depends on your system OS and CPU architecture. You can find full link to desired package in the release assets section below. 
+```sh
+wget https://github.com/ATOR-Development/ator-protocol/releases/download/<version>/<package_name>
+```
+2. Install package using apt
+```sh
+apt-get -y install ./anon_*.deb
```

Comment:
User may also need `apt update` on fresh os setup. Let's add it?

Reply:
Makes sense